docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -110,6 +110,28 @@ is used.
   * - ``iceberg.max-partitions-per-writer``
     - Maximum number of partitions handled per writer.
     - 100
+  * - ``iceberg.table-statistics-enabled``
+    - Enables :doc:`/optimizer/statistics`. The equivalent
+      :doc:`catalog session property </sql/set-session>`
+      is ``statistics_enabled`` for session specific use.
+      Set to ``false`` to disable statistics. Disabling statistics
+      means that :doc:`/optimizer/cost-based-optimizations` can
+      not make smart decisions about the query plan.
+    - ``true``
+
+ORC format configuration
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+The following properties are used to configure the read and write operations
+with ORC files performed by the Iceberg connector.
```

Comment:
This is worded a little awkwardly. Maybe `used to configure read and write operations performed by the Iceberg connector on ORC files.`

Reply:
I agree with you, do I need to add a PR to modify it?

Reply:
No .. its fine..